### PR TITLE
Fixes: #267 SearchView UI enhancements

### PR DESCRIPTION
**Pull request by nitish1211**
Search has better and improved UI.

g_20170104_1727403

> Really sorry for a new PR I accidentally closed the older one.

### CODE REVIEW

**Comment by smarx** [comment]
*Automated message from Dropbox CLA bot*

@nitish1211, it looks like you've already signed the Dropbox CLA. Thanks!

**Comment by Sam1301** [comment]
Thanks @nitish1211 for working on this. The code looks fine to me :+1:  (pinging @kunall17 for final review)
Also I noticed there are repeated lines `searchView UI enhancements` in the commit message.

**Comment by kunall17** [comment]
After re-formating the whole project by bc5a25f make sure you have formatted the code you have sent! (Just format the code by ctrl+alt+L or from the menu) 
And reword the commit as it have repeated lines!

**Comment by saketkumar95** [comment]
**Note** `Ctrl` + `alt` +`L` is used to lock the screen in ubuntu. You can change it from setting or try this alternatively `Ctrl` + `alt` + `shift` + `L`  for a dialog for reformatting. :)

**Comment by kunall17** [comment]
@saketkumar95 yeah dude, you got the point :P 
that would work on windows, option + command + L for the mac

**Comment by kunall17** [comment]
@nitish1211 Can you fix the merge conflicts!

**Comment by Sam1301** [comment]
Hey @nitish1211 , if it helps I use `git pull --rebase` to resolve merge conflicts.

**Comment by nitish1211** [comment]
@kunall17 @Sam1301 

I've resolved the merge conflicts, formatted the code and changed the commit message.

> Sorry I was inactive for a while as I was travelling.

**Comment by Sam1301** [comment]
No worries, can you please remove the merge commit? We generally don't use merge commits in our project.

**Comment by nitish1211** [comment]
@Sam1301 If I remove the merge commit its giving merge conflicts how do I work around this?

**Comment by yadav-rahul** [comment]
@nitish1211 You need to squash both commits to one. I can help you if you face any problem in squashing commits.

**Comment by Sam1301** [comment]
Can we chat about this in chat.zulip.org android stream? I'd be happy to help there.

File: app/src/main/java/com/zulip/android/activities/ZulipActivity.java [hunk 1]
```diff
@@ -135,41 +126,49 @@
...
-    // row from the people
-    final int allPeopleId = -1;
-    public MessageListFragment currentList;
-    FloatingActionButton fab;
     private ZulipApp app;
+
     private boolean logged_in = false;
     private boolean backPressedOnce = false;
+    private boolean inSearch = false;
+
```
Comment (yadav-rahul) [inline app/src/main/java/com/zulip/android/activities/ZulipActivity.java]: @nitish1211 Is this space intentional?

**Comment by nitish1211** [comment]
@kunall17 @Sam1301 I think I've finally done it all right this time can you please review this.

**Comment by Sam1301** [comment]
@nitish1211 Nice work! The code looks fine to me :+1:  @kunall17 Can you please give this a final review?
Also, the first letter in the commit message is generally capitalized.

**Comment by nitish1211** [comment]
@kunall17 Done.

**Comment by kunall17** [comment]
@timabbott LGTM!

**Comment by timabbott** [comment]
@nitish1211 can you update the commit message to explain in more detail what this does?  
Check out the Zulip commit message guidelines for more details: https://zulip.readthedocs.io/en/latest/version-control.html#commit-messages

**Comment by nitish1211** [comment]
@timabbott Would this be fine?

**Comment by nitish1211** [comment]
@timabbott  Done. Is this fine?

**Comment by timabbott** [comment]
It's better, but (1) The `Fixes` should have a blank line before it, and (2) the rest of the commit message ideally would explain the bug in more detail and why this is the correct solution.

Run `git log` in zulip.git to see what good commit messages look like.

**Comment by timabbott** [comment]
I cleaned that up and merged this as 1229ae6afef8443f10d119edd6bf4ade1924d0f3; just copied the text from your original issue description.  Probably could have benefitted from more editing.  

Thanks @nitish1211!

(BTW, you may want to set your `GIT_AUTHOR_NAME` to not just be your github user ID).

**Pull request closed without merging**